specification/serialconsole/resource-manager/Microsoft.SerialConsole/preview/2018-07-01-preview/examples/DisableConsoleExample.json

```diff
@@ -0,0 +1,17 @@
+{
+    "parameters": {
+        "api-version": "2018-07-01-preview",
+        "subscriptionId": "00000000-00000-0000-0000-000000000000",
+        "default": "default"
+    },
+    "responses": {
+        "500": {
```

Comment:
The spec doen't have response `"500"` but it has response `"200"`. Please, provide example for both: `"200"` and `"default"` responses
https://github.com/Azure/azure-rest-api-specs/blob/9a03d2dbc9201acf4956476a1927fcef49b604eb/specification/serialconsole/resource-manager/Microsoft.SerialConsole/preview/2018-07-01-preview/serialconsole.json#L170